README.md

```diff
@@ -140,6 +140,7 @@ conn.SetConnMaxLifetime(time.Hour)
   - `br` - `0` (Best Speed) to `11` (Best Compression)
   - `zstd`, `lz4` - ignored
 * block_buffer_size - size of block buffer (default 2)
+* read_timeout - a duration string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix such as "300ms", "1s". Valid time units are "ms", "s", "m" (default 5m).
```

Comment:
I'd suggest adding a reference to `time.ParseDuration` function (https://pkg.go.dev/time#ParseDuration) so the user is aware of how we parse the string.

Reply:
I have simply adapted the text from `dial_timeout` but I'm happy to add a reference to ParseDuration.

Reply:
@rolinh let's leave it as it is. I will follow up on this later. Sorry for the confusion.